Flow.Launcher.Core/Plugin/PluginsLoader.cs

```diff
@@ -98,7 +98,7 @@ public static IEnumerable<PluginPair> DotNetPlugins(List<PluginMetadata> source)
                                    + (erroredPlugins.Count > 1 ? "plugins have " : "plugin has ")
                                    + "errored and cannot be loaded:";
 
-                Task.Run(() =>
+                _ = Task.Run(() =>
```

Comment:
I don't think it makes sense to use a Task.Run here. Would you please replace it with the notification class?

Reply:
Completely agree - but [`Notification`](https://github.com/Flow-Launcher/Flow.Launcher/blob/dev/Flow.Launcher/Notification.cs) is not accessible outside of `Flow.Launcher`. I can open a follow up task to refactor `Notification` into `Flow.Launcher.Infrastructure` as part of a different PR, if that works?

Reply:
I agree. Let's move to `Flow.Launcher.Infrastructure` to make it usable across the solution.